jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/list/AbstractMultiGridViewTest.java

```diff
@@ -182,7 +201,8 @@ public Void answer(InvocationOnMock invocationOnMock) throws Throwable {
     @Test
     public void testInitActionsColumn() {
         ColumnMeta columnMeta = getView().initActionsColumn();
-        assertEquals(false, columnMeta.isVisibleIndex());
+        assertEquals(false,
+                     columnMeta.isVisibleIndex());
```

Comment:
@nmirasch I think you have older version od Kie codestyle. The newer one does not change the parameters to two-liners.

Reply:
I have put the last version and the code formatting is keeping it in two lines.. ??? 
I'm using https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/ide-configuration/intellij-configuration/code-style/intellij-code-style_droolsjbpm-java-conventions.xml. Should I use a different one?

Reply:
@nmirasch no, that is expected. the difference is the new one doesn't change one line to two lines as we can see here.

Reply:
I can change that manually for this specific case, but my IDE reformats that situation in two columns, That's makes my IDE code formatting unreliable regarding your comment .I'm going to update the PR

Reply:
Interesting, mine doesn't. Anyway, I don't require you to do that, just thought you have some older version of codestyle...